hack/make-rules/verify.sh

```diff
@@ -21,15 +21,10 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
 source "${KUBE_ROOT}/cluster/lib/util.sh"
 
-if [ -n "${VERBOSE}" ]; then
```

Comment:
uggh.  How annoying that verify end up calling verify-govet, which itself
has a top-level make rule.  Blargh.

Fine.

@soltysh are all other concerns wrapped up (all VERBOSE) mess?

On Thu, Jul 21, 2016 at 2:48 AM, Joe Farrell <notifications@github.com>
wrote:

> In hack/make-rules/verify.sh
> <https://github.com/kubernetes/kubernetes/pull/29185#discussion_r71676774>
> :
>
> > @@ -21,15 +21,10 @@ set -o pipefail
> >  KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
> >  source "${KUBE_ROOT}/cluster/lib/util.sh"
> >
> > -if [ -n "${VERBOSE}" ]; then
>
> The default behavior is fine, but it may be best to remove it from these
> scripts to avoid confusion in the future (as it is no longer a valid option
> for "verify", if this PR is merged)
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/29185/files/d5fe828d6380e2a7ac6e48e94efb2aa50b1c7892#r71676774>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVFuVtnacXf4A_ZVxYv25Xhvo3Riyks5qX0BzgaJpZM4JPkJQ>
> .
>